packages/nimble-components/src/number-field/specs/IxD.md

```diff
@@ -0,0 +1,146 @@
+| Error status icon   | Appears when a user has entered invalid data in a field.                                         |
+| Error text          | Appears when a user has entered invalid data in a field.                                         |
+
+### Related Components
+
+- [Number Field](https://nimble.ni.dev/storybook/index.html?path=/docs/components-number-field--docs)
+
+## Design
+
+### Configuration
+
+Proposed attributes:
+
+| Attribute               | Description                                                                                                               |
+| ----------------------- | ------------------------------------------------------------------------------------------------------------------------- |
+| coerce-invalid          | Boolean - Whether to coerce invalid max/min values on blur.                                                               |
+| maximum-fraction-digits | Maximum number of digits after the decimal separator.                                                                     |
+| minimum-fraction-digits | Minimum number of fraction digits. Values with fewer digits will be right-padded with zeros to meet the specified length. |
+
+### Behavior
+
+#### Mouse Interactions
+
+Hovering over the field input indicates interactivity.
+
+Clicking on the field input focuses the input and places a caret. If no value
+exists, the caret is placed at the far left. Otherwise, it is placed where
+clicked.
+
+![mouse interactions](./spec-images/localized-number-input-cursor-click.png)
+
+#### Keyboard Interactions
+
+| Key           | Description                                                                                                                                      |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `ENTER`/`ESC` | No action.                                                                                                                                       |
+| `UP ARROW`    | (If `hide-step = false`) Increment the number by the step value. (If `hide-step = true`) Moves the caret to the beginning of the entered number. |
+| `HOME`        | Moves the caret to the beginning of the entered number.                                                                                          |
+| `DOWN ARROW`  | (If `hide-step = false`) Decrement the number by the step value. (If `hide-step = true`) Moves the caret to the end of the entered number.       |
+| `END`         | Moves the caret to the end of the entered number.                                                                                                |
+| _Numbers_     | (If focused) Adds number at cursor.                                                                                                              |
```

Comment:
For completeness we should add `-` and `+`. If you want you can specify the behavior if the user tries to enter it in an invalid location or just leave that unspecified. The current behavior seems to allow them to be typed anywhere and then they're either truncated or the entire input is rejected on blur, depending on where the invalid ones are placed.

Reply:
Added - & + for completeness